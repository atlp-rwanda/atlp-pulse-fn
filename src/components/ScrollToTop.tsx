/* eslint-disable */
import React, { useEffect } from 'react';
import { useLocation } from 'react-router';

const ScrollToTop = (props: any) => {
  const location = useLocation();
  useEffect(() => {
    /* istanbul ignore next */
    window.scrollTo(0, 0);
  }, [location]);

  return <>{props.children}</>;
};

export default ScrollToTop;
