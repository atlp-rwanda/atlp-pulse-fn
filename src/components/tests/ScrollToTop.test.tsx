import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ScrollToTop from '../../components/ScrollToTop';

describe('<ScrollToTop />', () => {
  it('Renders ScrollToTop', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <ScrollToTop />
        </MemoryRouter>,
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
