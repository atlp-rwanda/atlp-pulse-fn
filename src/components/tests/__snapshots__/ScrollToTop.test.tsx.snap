// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScrollToTop /> Renders ScrollToTop 1`] = `null`;
